src/chrome/content/rules/Switch.ch.xml

```diff
@@ -21,10 +23,14 @@
 	<target host="www.switch.ch" />
 	<target host="cms.www.switch.ch" />
 	<target host="misc.www.switch.ch" />
-
-
-	<securecookie host="^\.switch\.ch$" name=".+" />
-
+	<target host="filesender.switch.ch" />
+	<target host="drive.switch.ch" />
+	<target host="cloud-id.switch.ch" />
+	<target host="portfolio.switch.ch" />
+	<target host="tube.switch.ch" />
+	<target host="webcam.switch.ch" />
```

Comment:
See also:
```
aai.switch.ch (m)
attribute-viewer.aai.switch.ch
metadata.aai.switch.ch (m)
rr.aai.switch.ch
forum.cloud.switch.ch
sandstorm.cloud.switch.ch
cast.switch.ch
player.cast.switch.ch
cloudblog.switch.ch
collab.switch.ch
engines.switch.ch
identityblog.switch.ch
interact.switch.ch
forge.switch.ch
frisal.switch.ch (t)
gitlab.switch.ch
lists.switch.ch
ndt.switch.ch (r)
kb.pert.switch.ch (m)
projects.switch.ch
securityblog.switch.ch
toolbox.switch.ch

Reply:
Please have a look at our docs: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains
You should also sort the targets alphabetically.

Reply:
I've addes some more targets and sorted them.